src/main/java/io/neonbee/internal/helper/FileSystemHelper.java

```diff
@@ -99,28 +102,55 @@ private FileSystemHelper() {}
         return readFile(vertx, path).map(Buffer::toJsonObject);
     }
 
+    /**
+     * This method reads the entire file in JSON format synchronously and converts it to a {@link JsonObject}.
+     *
+     * @param path The path of the file
+     * @return a future to the converted {@link JsonObject}
+     * @throws IOException if an I/O error occurs reading from the file or a malformed or unmappable byte sequence is
+     *                     read
+     */
+    public static JsonObject readJSON(Path path) throws IOException {
+        return new JsonObject(Files.readString(path, UTF_8));
+    }
+
     /**
      * This method reads the entire file in YAML format asynchronously and converts it to a {@link JsonObject}.
      *
      * @param vertx The related Vert.x instance
      * @param path  The path of the file
-     * @return a future to the converted {@link JsonObject}
+     * @return the {@link JsonObject}
      */
     public static Future<JsonObject> readYAML(Vertx vertx, Path path) {
         return readFile(vertx, path).compose(buffer -> parseYAML(vertx, buffer));
     }
 
+    /**
+     * This method reads the entire file in YAML format synchronously and converts it to a {@link JsonObject}.
+     *
+     * @param path The path of the file
+     * @return the YAML file converted to an {@link JsonObject}
+     * @throws IOException if an I/O error occurs reading from the file or a malformed or unmappable byte sequence is
+     *                     read
+     */
+    public static JsonObject readYAML(Path path) throws IOException {
+        return parseYAML(Files.readAllBytes(path));
+    }
+
     private static Future<JsonObject> parseYAML(Vertx vertx, Buffer buffer) {
         return vertx.executeBlocking(promise -> {
```

Comment:
Use executeBlocking from AsyncHelper?

Reply:
The `AsyncHelper.executeBlocking(...)` takes a vertx instance which we cannot provide when we calling the method.